contracts/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "@dcgrants/contracts",
+  "version": "0.0.1",
+  "dependencies": {},
+  "devDependencies": {
+    "@ethersproject/abstract-signer": "^5.0.6",
+    "@ethersproject/bignumber": "^5.0.8",
```

Comment:
Recommend removing these since they are included with `ethers` so we don't need them here, and should also bump the ethers dependency below to 5.4.0 which adds EIP-1559 support